lib/CSSStyleDeclaration-impl.js

```diff
@@ -123,49 +128,32 @@ CSSStyleDeclaration.prototype = {
 
     this._onChange(this.cssText);
     return prevValue;
-  },
+  }
 
   /**
    *
    * @param {String} name
    */
-  getPropertyPriority: function(name) {
+  getPropertyPriority(name) {
     return this._importants[name] || '';
-  },
-
-  getPropertyCSSValue: function() {
-    //FIXME
-    return;
-  },
-
-  /**
-   *   element.style.overflow = "auto"
-   *   element.style.getPropertyShorthand("overflow-x")
-   *   -> "overflow"
-   */
-  getPropertyShorthand: function() {
-    //FIXME
-    return;
-  },
-
-  isPropertyImplicit: function() {
-    //FIXME
-    return;
-  },
+  }
 
   /**
    *   http://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSStyleDeclaration-item
    */
-  item: function(index) {
-    index = parseInt(index, 10);
+  item(index) {
     if (index < 0 || index >= this._length) {
       return '';
     }
     return this[index];
-  },
-};
+  }
+
+  [idlUtils.supportsPropertyIndex](index) {
+    return index >= 0 && index < this._length;
+  }
+}
 
-Object.defineProperties(CSSStyleDeclaration.prototype, {
+Object.defineProperties(CSSStyleDeclarationImpl.prototype, {
```

Comment:
Changing this to be defined as getters in the class body pushed this PR over the edge of what git will consider a renamed file.

Reply:
Don't let tools dictate your code :) Worst case we could just turn on `-w` and `-M` to make Git try harder.